core/chains/evm/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -151,7 +153,7 @@ func (b *BulletproofTxManager) Start() (merr error) {
 			b.logger.Warnf("Chain %s does not have any eth keys, no transactions will be sent on this chain", b.chainID.String())
 		}
 
-		eb := NewEthBroadcaster(b.db, b.ethClient, b.config, b.keyStore, b.eventBroadcaster, keyStates, b.gasEstimator, b.resumeCallback, b.logger)
+		eb := NewEthBroadcaster(b.db, b.ethClient, b.config, b.keyStore, b.eventBroadcaster, keyStates, b.gasEstimator, b.resumeCallback, b.logger, b.checkerFactory)
```

Comment:
Nit: param list really long, would be nice if it was broken up into multiple formatted lines.

Reply:
^